crates/apollo-compiler/src/database/hir_db.rs

```diff
@@ -67,6 +68,9 @@ pub trait HirDatabase: InputDatabase + AstDatabase {
         name: String,
     ) -> Option<Arc<OperationDefinition>>;
 
+    /// Return an operation definition without a name corresponding to the file id.
+    fn find_unnamed_operation(&self, file_id: FileId) -> Option<Arc<OperationDefinition>>;
```

Comment:
This is a drive-by new feature. Should it be called `find_anonymous_operation` instead? The spec uses both terms

Reply:
jinx! i like `anonymous` better